cmd/microcloud/cluster_members.go

```diff
@@ -141,3 +174,75 @@ func (c *cmdClusterMemberRemove) Run(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+type cmdClusterEdit struct {
+	common *CmdControl
+}
+
+func (c *cmdClusterEdit) command() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "edit",
```

Comment:
I think this command sounds a bit too benign. Perhaps we should call it `recover`?

Reply:
I'll go with recover since that's what we've been using amongst ourselves; `reconfigure` could work too (as used internally in LXD) if we're worried about it not sound malignant enough